doc/sneak.txt

```diff
@@ -259,6 +259,13 @@ OPTIONS                                                        *sneak-options*
 To change an option, add a |let| statement in your |vimrc|. Example: >
     let g:sneak#use_ic_scs = 0
 <
+g:sneak#no_default_mappings = 0
```

Comment:
flip it so it is "positive logic"

```suggestion
g:sneak#default_mappings = 1
```

Reply:
I prefer the negative one here because it sounds natural for me (at least) to be set to 0 as default, and user will set it to 1 to turn on especially when an option expects either 0 or 1 as "OFF" or "ON".

Addition to that, I usually unify default values to be set to 0 unless the variable has specific reason to be other than 0; such unification reduces unnecessary chances that users should grep default values.

However, I also know positive logic is preferable in most cases so that it's a matter of taste after all. Please adopt the one on your preference.